File: 080-C++/CppClassExample/src/Student.cpp
#include <iostream>
#include <string>

using namespace std;

class Student {
  private:
    string name;

  public:
    Student(string name);
    string getName();
    void setName(string name);

};

Student::Student(string name){
  this->name = name;
}

string Student::getName(){
  return name;
}

void Student::setName(string name){
  this->name = name;
}

int main(void){
  Student student("Tester");
  cout << "Name of Student: " << student.getName() <<endl;
  return 0;
}

File: 070-C#/010-Net/020-Mono/MonoHelloWorldExample/src/Student.cs
using System;

public class Student{

  private string name;

  public Student(string name){
    this.name = name;
  }

  public string getName(){
    return name;
  }

  public void setName(string name){
    this.name = name;
  }

  static public void Main(string[] args){
      Console.WriteLine("Hello World! This is Tech Office C# Example.");
      Student student = new Student("Tester");
      Console.WriteLine("The name of Student: " + student.getName());
  }

}
